extra_geom/tests/test_crystfel_geometry.py

```diff
@@ -0,0 +1,86 @@
+from extra_geom import GenericGeometry
```

Comment:
Don't foget to rename this test file.

Also, as we're leaving reading `.geom` files for a separate PR, let's leave the files for that test until we need them. :)

Reply:
I totally forgot about this, thanks! Done.